apis/core/v1alpha2/flagsourceconfiguration_types.go

```diff
@@ -59,6 +59,11 @@ type FlagSourceConfigurationSpec struct {
 	// DefaultSyncProvider defines the default sync provider
 	// +optional
 	DefaultSyncProvider string `json:"defaultSyncProvider"`
+
+	// LogFormat allows for the sidecar log format to be overridden, defaults to 'json'
+	// +optional
+	// +kubebuilder:default:=json
```

Comment:
will this cause issues when we merge the CRs? currently we are checking if `new.LogFormat != ""`, if this always sets a default value of `"json"` we will override configurations with this default on each merge

Reply:
i can also remove it again. i saw that you don't use it for anything else

Reply:
my vote would be to remove it, but happy to update the merge function to check for non-default

Reply:
i removed it :)